content/topics/git/version-control-and-scrum/_index.md

```diff
@@ -66,8 +87,7 @@ weight: 15
 
 ## General issue board procedure
 
-Every time you move a ticket from one column to another, you are unassigning yourself. A ticket in the done column is reassigned to the person who did the development work on it.
-Comment on the tickets as a form of taking notes & passing on explanations from one person to another.
+Every time you move a ticket from one column to another, you make a PR and assign a team member for the first review on your code. If the code has no bugs or errors then the ticket moves to final code review so it can be merged.If there are any changes requested comments are made on the PR and then the ticket is moves to reveiw changes requested. Tickets that are merged move to the Done column.
```

Comment:
Add: 

"""
Please note: The columns described here are useful for this discussion, and if you are new to Scrum then these columns will work well for you. There isn't a rule that says that you need to name your columns exactly like this. Please make sure you understand the general procedure outlined here, then adapt it for your team
"""

Reply:
@tspmod the content of this comment from Sheena still seems to be missin